storybookjs/storybook
# Angular: Fix nx project with workspace.json

Issue:
https://github.com/storybookjs/storybook/commit/4f004183eafc9be7cf404b6dc2b1a22801e86201#r51674942
https://github.com/nrwl/nx/issues/5688#issuecomment-853542895


## What I did




separate the commit of this [PR](https://github.com/storybookjs/storybook/pull/15184) to fix the bug for nx as soon as possible.
it is finally not linked to angular 12


## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
